lib/tasks/panopticon.rake

```diff
@@ -85,4 +85,22 @@ namespace :panopticon do
       end
     end
   end
+
+  desc "Re-register all content with Panopticon"
+  task :re_register_all_content => :environment do
+    logger = GdsApi::Base.logger = Logger.new(STDERR).tap { |l| l.level = Logger::INFO }
+
+    published_editions = Edition.all
```

Comment:
Thank for the feedback!

1. It is, I copied the code from the script above and forgot to rename it. Will do.
2. Great thinking, will do this too.